Reference/Querying/IPublishedContent/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-versionFrom: 7.0.0
+versionFrom: 8.0.0
 needsV8Update: "true"
```

Comment:
```suggestion
```
This yaml tag is for docs pages that needs to be verified they work for v8. If we have a v8 version then it will no longer be needed 🙂